contrib/terraform/.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform/
+*.tfvars
```

Comment:
One thing we may want to do:

Make the TF templates completely ignorant of the particular OpenStack environment they're deployed to (i.e., no references to anything Scality Cloud specific), and ship a `scality-cloud.tfvars` file which provides the 'defaults' for our particular cloud environment (i.e., networks, machine types, VM images,...), and document how to deploy in SC (by using `-var-file` when invoking `terraform apply`).